server/src/db/migrations/20230516173200_addUserRole.cjs

```diff
@@ -0,0 +1,21 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+    return knex.schema.table("users", (table) => {
+        table.string("role").notNullable().defaultTo("student")
```

Comment:
depending on the functionality you're looking for with roles, using an `enum` might be a better way to maintain different roles (student, admin, etc.)